packages/aggregate-api/test/aggregate.js

```diff
@@ -174,31 +145,22 @@ export const test = {
       channel: createServer(context),
     })
 
-    // Generate CAR Files for offer
-    const offers = (await randomCargo(100, 128))
-      // Inflate size for testing above range
-      .map((car) => ({
-        ...car,
-        size: car.size * 10e6,
-      }))
-    const size = offers.reduce((accum, offer) => accum + offer.size, 0)
-    const piece = {
-      link: parseLink(
-        'baga6ea4seaqm2u43527zehkqqcpyyopgsw2c4mapyy2vbqzqouqtzhxtacueeki'
-      ),
-      size,
-    }
+    // Generate Pieces for offer
+    const { pieces, aggregate } = await randomAggregate(100, 128)
+    const badHeight = 31
+    const size = 2n ** BigInt(badHeight) * BigInt(Node.Size)
```

Comment:
We already have [PaddedSize.toHeight](https://github.com/web3-storage/data-segment/blob/61ee74d4ebc5c8eec8b5d4dcea2b46e844fe1dc8/src/piece/padded-size.js#L55-L60) we probably should just add `PaddedSize.fromHeight` so we don't have to do this math all over the place.

Reply:
https://github.com/web3-storage/data-segment/pull/18